lib/sequenceserver/blast.rb

```diff
@@ -104,6 +103,10 @@ def pre_process(params)
         params[:sequence].strip! unless params[:sequence].nil?
       end
 
+      def blast_formatter(rfile_path, ofmt, specifiers = nil)
+        `blast_formatter -archive '#{rfile_path}' -outfmt '#{ofmt} #{specifiers}'`
```

Comment:
Fair enough. Add a note in the source code though.

What's the behaviour with XML?